src/sanitizer.js

```diff
@@ -186,9 +190,19 @@ export function sanitizeHtml(html) {
         }
         return;
       }
+      const bindAttribsIndices = map();
```

Comment:
That's what I started with. Will recommended I use an object instead, and then Justin recommended using map.